server/http.go

```diff
@@ -87,17 +100,21 @@ func (p *Plugin) handleStandardWebhook(w http.ResponseWriter, r *http.Request, w
 	post.Message = "Meeting has ended."
 	post.Props["meeting_status"] = zoom.WebhookStatusEnded
 
-	if _, appErr := p.API.UpdatePost(post); appErr != nil {
+	_, appErr = p.API.UpdatePost(post)
+	if appErr != nil {
 		http.Error(w, appErr.Error(), appErr.StatusCode)
 		return
 	}
 
-	if appErr := p.API.KVDelete(key); appErr != nil {
+	appErr = p.API.KVDelete(key)
+	if appErr != nil {
 		p.API.LogWarn("failed to delete db entry", "error", appErr.Error())
 		return
 	}
 
-	if _, err := w.Write([]byte(post.ToJson())); err != nil {
+	w.WriteHeader(http.StatusOK)
```

Comment:
Is this needed?

Reply:
It's not necessary. Thank you for your explanation https://github.com/mattermost/mattermost-plugin-github/pull/239#discussion_r422431624